src/SWP/Bundle/CoreBundle/EventListener/IntroduceYourselfListener.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2018 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2018 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use SWP\Component\Storage\Factory\FactoryInterface;
+
+final class IntroduceYourselfListener
```

Comment:
This class name is weird to me and no idea what that class does exactly. Why not name it `CustomPublisherHeaderListener` or `SetCustomPublisherHeaderListener`